README.md

```diff
@@ -32,6 +32,15 @@ GdsApi.configure do |config|
 end
 ```
 
+By default, both `always_raise_for_not_found` and `hash_response_for_requests`
```

Comment:
I might even remove this message from the README and only leave the deprecation warnings. WDYT?